src/rgw/rgw_lua_background.cc

```diff
@@ -177,5 +189,46 @@ void Background::create_background_metatable(lua_State* L) {
   create_metatable<rgw::lua::RGWTable>(L, true, &rgw_map, &table_mutex);
 }
 
+#ifdef WITH_RADOSGW_LUA_PACKAGES
+int Background::PackagesWatcher::start() {
+  return static_cast<rgw::sal::RadosLuaManager*>(parent->lua_manager.get())->watch_reload(&parent->dp, &watch_handle, this);
```

Comment:
this is ugly...
see comment on: https://github.com/ceph/ceph/pull/52326/files#diff-8d5ab77cf6b24e95c2c7b352c52d1e254781f6c9e7fb3fd148bddc8a3f4321e7R3480

Reply:
Ceph code, like comedy, is not pretty ;)